pkg/crc/validation/validation.go

```diff
@@ -51,3 +52,36 @@ func ValidateBundle(bundle string) error {
 	}
 	return nil
 }
+
+// ValidatePath check if provide path is exist
+func ValidatePath(path string) error {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
@cfergeau this is the file downloaded by the user or created by him/her, this is not the file which will be present already on system so I choose not to check, also when you try to read it using `ioutil` it will through the error if permission is not correct.